cmd/root.go

```diff
@@ -53,8 +53,8 @@ func init() {
 	// version subcommand: nothing but the version.
 	root.SetVersionTemplate(`{{printf "%s\n" .Version}}`)
 
-	// Prefix all environment variables with "FUNC_" to avoid collisions with other apps.
-	viper.SetEnvPrefix("func")
+	// Prefix all environment variables with "FUNCTION_" to avoid collisions with other apps.
+	viper.SetEnvPrefix("function")
```

Comment:
`func` ?

Reply:
thanks. Gotchas while resolving the conflicts.